cfpb/cfgov-refresh
# Add meta descriptions to prepaid agreement pages

Add meta and Open Graph descriptions to individual prepaid agreement pages instead of the default description ("Our vision is a consumer finance marketplace that works for American consumers, responsible providers, and the economy as a whole.") that shows up on those pages now.

## Additions

- `desc` and `og_desc` blocks to the prepaid agreement detail page template


## Testing

1. Fire up this branch and visit http://localhost:8000/data-research/prepaid-accounts/search-agreements/
2. Go to the detail page of any agreement listed there
3. View source on that page
4. Confirm that the meta and Open Graph descriptions are like "Download the most recent prepaid product agreement files for the `{{ product.name }}` issued by `{{ product.issuer_name }}`." where the `{{ product.name }}` and `{{ product.issuer_name }}` bits match up with the info on the page. So, for agreement [45540](http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/45540/), for example, the description should say "Download the most recent prepaid product agreement files for the My Target Paycard issued by Sunrise Banks, National Association."

## Notes

- See GHE/CFGOV/platform/issues/3727 for the background on this one
- Some agreement detail pages won't have a totally unique description. There are some agreements that have the same `{{ product.name }}` and `{{ product.issuer_name }}` but differ in the date the agreement was submitted or in one of the other fields. Those seem like outliers, though, and this change will give the vast majority of the agreement detail pages a unique description (and even ones with duplicate descriptions will still have a good, informative description).
- I chatted with @sonnakim about the wording of the description, and she thought it sounded good. @virginiacc, how does that look to you as someone from the original team who knows a lot more about these agreements than I do?

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
